prow/kube/metrics.go

```diff
@@ -118,6 +123,15 @@ func GatherProwJobMetrics(current []prowapi.ProwJob) {
 	// stale metrics like triggered or pending jobs that are now complete.
 	prowJobs.Reset()
 
+	// record prow version
+	version, err := version.VersionTimestamp()
+	if err != nil {
+		// Not worth panicking
```

Comment:
You don't need to panic, but please log the error.

Reply:
done